fortitude/src/rules/mod.rs

```diff
@@ -74,92 +74,92 @@ pub fn code_to_rule(category: Category, code: &str) -> Option<(RuleGroup, Rule)>
-        (Precision, "011") => (RuleGroup::Stable, Ast, precision::double_precision::DoublePrecision),
-        (Precision, "021") => (RuleGroup::Stable, Ast, precision::implicit_kinds::ImplicitRealKind),
-
-        (Modules, "001") => (RuleGroup::Stable, Ast, modules::external_functions::ProcedureNotInModule),
-        (Modules, "011") => (RuleGroup::Stable, Ast, modules::use_statements::UseAll),
-        (Modules, "012") => (RuleGroup::Preview, Ast, modules::use_statements::MissingIntrinsic),
-        (Modules, "021") => (RuleGroup::Preview, Ast, modules::accessibility_statements::MissingAccessibilityStatement),
-        (Modules, "022") => (RuleGroup::Preview, Ast, modules::accessibility_statements::DefaultPublicAccessibility),
-        (Modules, "031") => (RuleGroup::Preview, Ast, modules::include_statement::IncludeStatement),
-        (Modules, "041") => (RuleGroup::Preview, Ast, modules::file_contents::MultipleModules),
-        (Modules, "042") => (RuleGroup::Preview, Ast, modules::file_contents::ProgramWithModule),
-
-        (Io, "001") => (RuleGroup::Preview, Ast, io::missing_specifier::MissingActionSpecifier),
-        (Io, "011") => (RuleGroup::Preview, Ast, io::magic_io_unit::MagicIoUnit),
-        (Io, "012") => (RuleGroup::Preview, Ast, io::magic_io_unit::NonPortableIoUnit),
-
-        (Readability, "001") => (RuleGroup::Preview, Ast, readability::magic_numbers::MagicNumberInArraySize),
-
-        (Bugprone, "001") => (RuleGroup::Preview, Ast, bugprone::select_default::MissingDefaultCase),
-        (Bugprone, "011") => (RuleGroup::Preview, Ast, bugprone::trailing_backslash::TrailingBackslash),
+        (Error, "000") => (RuleGroup::Stable, None, Default, error::ioerror::IoError),
+        (Error, "001") => (RuleGroup::Stable, Ast, Default, error::syntax_error::SyntaxError),
+        (Error, "011") => (RuleGroup::Stable, None, Default, error::allow_comments::InvalidRuleCodeOrName),
+
+        (Filesystem, "001") => (RuleGroup::Stable, Path, Default,filesystem::extensions::NonStandardFileExtension),
+
+        (Style, "001") => (RuleGroup::Stable, Text, Default, style::line_length::LineTooLong),
+        (Style, "021") => (RuleGroup::Stable, Ast, Default, style::exit_labels::MissingExitOrCycleLabel),
+        (Style, "041") => (RuleGroup::Stable, Ast, Default, style::old_style_array_literal::OldStyleArrayLiteral),
+        (Style, "051") => (RuleGroup::Stable, Ast, Default, style::relational_operators::DeprecatedRelationalOperator),
+        (Style, "061") => (RuleGroup::Stable, Ast, Default, style::end_statements::UnnamedEndStatement),
+        (Style, "071") => (RuleGroup::Stable, Ast, Default, style::double_colon_in_decl::MissingDoubleColon),
+        (Style, "081") => (RuleGroup::Preview, Ast, Default, style::semicolons::SuperfluousSemicolon),
+        (Style, "082") => (RuleGroup::Preview, Ast, Default, style::semicolons::MultipleStatementsPerLine),
```

Comment:
I'm not sure if this one should be on by default. `superfluous-semicolon` should definitely be on though.